client/src/components/table/index.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import './style.css'
+class Table extends React.Component{
+  constructor(props){
+    super(props);
+    this.state={  
+      flightNo :'',
+      flightDate:'',
+      flightTime:'',
+      flightDuration:'',
+      flightsArray:[]
+    }
+    this.handleChange=this.handleChange.bind(this);
+    this.flightDate=this.flightDate.bind(this);
+    this.flightTime=this.flightTime.bind(this);
+    this.flightDuration=this.flightDuration.bind(this);
+  } 
+  
+  handleChange=(e)=>{
+    this.setState({flightNo:e.target.value})
+    this.flightDate();
+    this.flightTime();
+    this.flightDuration();
+  }
+
+  
+  flightDate=(e)=>{
+    this.setState({flightDate:e.target.value})
+  }
+  flightTime=(e)=>{
+    this.setState({flightTime:e.target.value})
+  }
+
+  flightDuration=(e)=>{
+    this.setState({flightDuration:e.target.value})
+  }
+    render(){
+  
+
+      return(
+        
+
+      <div className ="rr">
+          <table id ="table"   onChange={(event)=> this.handleChange(event)}>
+            <tbody>
+              <tr>
+                <th>flight no.</th>
```

Comment:
I know this is temporary code and will change once you're displaying flights from the server, but in the future you could try replacing the `<table>` and related tags (td, tr, ...) with divs and use flexbox to style the table.

Then reach row would have `display: 'flex'`, and you would have a lot of control with how to align the columns and position everything.

Reply:
@razan99 I already told you to change that, and to handle the table's data from the state until we're calling it from the server.